vim/settings/vimshell.vim

```diff
@@ -0,0 +1 @@
+map ,s :sp<CR>:resize 10<CR>:VimShell<CR
```

Comment:
sorry to keep making you change things but this keymap will interfere with other ones starting with `,s` (to see the list, use `:map ,s`

How about we make this `,vs` for vimshell